### ISSUE TITLE
No Photos Found

### ISSUE THREAD

**Issue by UnPossible-Me** [photoprism/photoprism#1943]
Hello!
Pretty much what the title says, indexing happens quite quickly, & subsequently, no photos are found. 
After going though previous issues,  [#777](https://github.com/photoprism/photoprism/issues/777), & posts online, I'd hazard a guess as to say it's likely permission issues.
The thing is, how do I make it work?

I'm running PP from docker, on a Raspberry Pi alongside Nextcloud using guides given in the docs. Everything seems to startup just fine.
My `docker-compose.yml` looks like this:
```
      HOME: "/mnt/HDD1/photoprism"
    ## Hardware devices for video transcoding and machine learning (optional):
    # devices:
    #  - "/dev/video11:/dev/video11" # Video4Linux (h264_v4l2m2m)
    working_dir: "/mnt/HDD1/photoprism"
    volumes:
      ## The *originals* folder contains your original photo and video files (- "[host folder]:/photoprism/originals"):
      - "/mnt/HDD1/nextcloud/data/UnPossible/files/Photos:/photoprism/orginals"
      ## Multiple folders can be made accessible by mounting them as subfolders of /photoprism/originals:
      # - "/mnt/Family:/photoprism/originals/Family"    # [folder 1]:/photoprism/originals/[folder 1]
      # - "/mnt/Friends:/photoprism/originals/Friends"  # [folder 2]:/photoprism/originals/[folder 2]
      ## You may mount an *import* folder from which files can be transferred to *originals* (optional):
      # - "~/Import:/photoprism/import"
      ## Cache, session, thumbnail, and sidecar files will be created in the *storage* folder (never remove):
      - "./storage:/mnt/HDD1/photoprism/storage"
```

Now, all my `nextcloud` files are owned by `www-data` with `755` permissions.
I'm running `docker-compose up -d` as `root`, so I was hoping photoprism would be able to access these files as well.

A trick I read was to launch a console as root in the PP container & check if it can find the photos. 
Well, I can't the photoprism directory is all that's available. 

I'm aware of mounting Nextcloud folders from `sync` settings, but that just copies them over to the originals folder AFAIK, nice for backups, but useless since they're both running on the same machine & I don't have the luxury of hosting 2 copies. 

Any ideas? Or am I just missing something obvious?

**Comment by lastzero** [comment]
Never use GitHub Issues if you have a general question, need help, or an issue has not (yet) been clearly identified. Thank you! 💐

You are welcome to use GitHub Discussions instead:
https://github.com/photoprism/photoprism/discussions

Sponsors receive direct technical support via email:
https://photoprism.app/contact 📬

**What does not work as expected?**

Our troubleshooting checklists help you quickly find and fix common problems:
https://docs.photoprism.app/getting-started/troubleshooting/

**How can we reproduce it?**

Steps to reproduce the behavior:

1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error

If reporting an Import, Indexing, or Performance issue, please include the number and type of pictures in your library,
as well as any configuration options you have changed e.g. for thumbnail quality.

**What behavior do you expect?**

A clear and concise description of what you expected to happen.

**What could be the cause of your problem?**

Try to determine the cause of your problem before submitting a bug report:
https://docs.photoprism.app/getting-started/troubleshooting/

**Can you provide us with example files for testing, error logs, or screenshots?**

Please add example files or screenshots that help to reproduce your problem.
You may also send files via email or share a download link:
https://photoprism.app/contact

Learn how to diagnose frontend issues:
https://docs.photoprism.app/getting-started/troubleshooting/browsers/

IMPORTANT: Always provide a sample file and logs if it is an Import, Indexing, or Metadata issue, otherwise we will not be able to process your report.

**Which software versions do you use?**

- PhotoPrism Architecture & Build Number (AMD64, ARM64, ARMv7,...)
- Database Type & Version (MariaDB, MySQL, SQLite,...)
- Operating System Types & Versions (Linux, Windows, Android,...)
- Browser Types & Versions (Firefox, Chrome, Safari on iPhone,...)
- Browser Plugins & Ad Blockers (if any)

NOTE: The app version / build number can be found in *Settings* when you scroll to the bottom. MySQL 8 is not officially supported anymore, see FAQs and Release Notes.

IMPORTANT:
1. Always provide database and operating system details if it is a Backend, Import, or Indexing issue.
2. Always provide web browser and operating system details if it is a Frontend issue.
3. If it is a Performance problem, you must provide ALL information.

**On what kind of device is PhotoPrism installed?**

This is especially important if you are reporting a Performance, Import, or Indexing problem. You can skip this if you are reporting a problem found on our public demo, or if it is a completely unrelated issue, such as a broken page layout.

- Device / Processor Type (Raspberry Pi 4, Intel Core i7-3770, AMD Ryzen 7 3800X,...)
- Physical Memory & Swap Space (in GB)
- Storage Type (HDD, SSD, RAID, USB, Network Storage,...) 
- anything else that might be helpful

IMPORTANT: Always provide device, memory, and storage details when you have a Backend, Performance, Import, or Indexing issue.

**Do you use a Reverse Proxy, Firewall, VPN, or CDN?**

If yes, please specify type and version. You can skip this if you are reporting a completely unrelated issue.

IMPORTANT:
1. Always provide this information when you have a Reliability, Performance, or Frontend problem, such as failed uploads, connection errors, broken thumbnails, or video playback issues.
2. If you are using NGINX, also provide its configuration and/or consider asking the NGINX community for advice as we do not specialize in supporting their product: https://docs.photoprism.app/getting-started/proxies/nginx/

# INCOMPLETE REPORTS CANNOT BE PROCESSED ☹️ #

**Comment by lastzero** [comment]
### Missing Pictures ###

If you have indexed your library and some images or videos are missing, first [check *Library > Errors* for errors and warnings](logs.md).
In case the application logs don't contain anything helpful:

- [ ] The pictures are in [Review](../../user-guide/organize/review.md) due to low quality or incomplete metadata
- [ ] The [file type](https://docs.photoprism.app/getting-started/faq/#what-media-file-types-are-supported) is generally unsupported
- [ ] The [file type](https://docs.photoprism.app/getting-started/faq/#what-media-file-types-are-supported) is generally supported, but a specific feature or codec is not
- [ ] The files have bad filesystem permissions, so they can't be opened by the indexer
- [ ] The indexer has skipped the files because they are exact duplicates
- [ ] The files are ignored based on pattern in a `.ppignore` file
- [ ] They are in *Library > Hidden* because thumbnails could not be created:
    - [ ] *Convert to JPEG* is disabled in *Settings > Library*
    - [ ] FFmpeg and/or RAW converters are disabled in *Settings > Advanced*
    - [ ] The file is broken and cannot be opened
    - [ ] The sidecar and/or cache folders are not writable
- [ ] Multiple files were stacked based on their metadata or file names
- [ ] The private or archived status was restored from a backup
- [ ] The NSFW filter is enabled, so they were marked as private
- [ ] You are not signed in as admin, so you can't see everything
- [ ] You try to index a shared drive on a remote server, but the server is offline
- [ ] The indexer has crashed because you didn't configure [at least 4 GB of swap](https://docs.photoprism.app/getting-started/troubleshooting/docker/#adding-swap)
- [ ] Somebody has deleted files without telling you
- [ ] You are connected to the wrong server, VPN, CDN, or a DNS record has not been updated yet

Start a rescan once the issue has been resolved.

**Comment by UnPossible-Me** [comment]
Thanks for quick response, if this is an issue that's to be moved over to the discussions, I'll gladly shift.
In the meanwhile,  I'll attempt to give more information:

I'm on a Raspberry Pi 4, 4G RAM with 4G of swap. (ARMv8 64bit version)
Using MariaDB v10.6 as my database.
Photoprism 211215-93b26f19-Linux-aarch64
All of it running though Docker,
Currently with NGINX as the webserver, locally without a reverse proxy.

All the data is stored on an external HDD. (`/mnt/HDD1`)
Now, all my pictures, around a thousand for the initial run, are stored as `.jpg` in subfolders of the `/mnt/HDD1/nextcloud/data/UnPossible/files/Photos` directory specified in the config file.

All the log files are empty, `level=info msg="indexing completed in 0 s"`
No errors, no info, nothing.
I've gone over the checklist you've sent, no dice.

I've tried manually uploading a file annnnnnd it works!
It uploads, indexes, & shows up in a new folder.
Unfortunately, I cannot for the life of me find where it's located. I've checked all directories even remotely related to PP, used `find` to try & locate the original name & new one, no luck. 

Any idea on how to proceed?
